<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeighborhoodStatisticsClass</class>
 <widget class="QDialog" name="NeighborhoodStatisticsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeighborhoodStatistics</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>54</y>
     <width>321</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="openRasterLabel">
        <property name="text">
         <string>输入栅格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inputCombo">
        <property name="minimumSize">
         <size>
          <width>211</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openPushButton">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>23</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="TigerGIS.qrc">
          <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="outputRasterLabel">
        <property name="text">
         <string>输出栅格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="outputPushButton">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>23</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="TigerGIS.qrc">
          <normaloff>:/TigerGIS/res/folderOpen.png</normaloff>:/TigerGIS/res/folderOpen.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="heightLabel">
        <property name="text">
         <string>高    度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="heightLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="widthLabel">
        <property name="text">
         <string>宽    度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="widthLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="typeLabel">
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>统计类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>MEAN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MAXIMUM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MINIMUM</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="surePushButton">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TigerGIS.qrc"/>
 </resources>
 <connections/>
</ui>
